### PR TITLE
Added optional select-all and deselect-all boxes in TableViewCol

### PR DESCRIPTION
**Pull request by WillPeers**

### CODE REVIEW

**Comment by WillPeers** [comment]
LGTM great work

File: src/components/TableViewCol.js [hunk 1]
```diff
@@ -85,6 +85,54 @@ export default function TableViewCol({ columns, groupedColumns, onColumnUpdate,
         />
       )}
       <FormGroup className={classes.formGroup}>
+      {options.selectAll && (
```
Comment (vbruzzi) [inline src/components/TableViewCol.js]: Can you document the new options and props on the README?

File: src/components/TableViewCol.js [hunk 2]
```diff
@@ -85,6 +85,54 @@ export default function TableViewCol({ columns, groupedColumns, onColumnUpdate,
...
+              }}
+              onChange={() => onSelectAllColumns()}
+              checked={columns.every(column => column.display === 'true')}
+              value="selectAll"
+            />
+          }
+          label='Select All'
+        />
+        )}
+        {options.deselectAll && (
```
Comment (vbruzzi) [inline src/components/TableViewCol.js]: Right now if we have both `deselectAll` and `selectAll` will both checkboxes show?

**Review by vbruzzi — APPROVED** [review]
LGTM

**Pull request merged**